src/eko/runner/compute.py

```diff
@@ -0,0 +1,8 @@
+"""Compute parts from recipes."""
+from .parts import Part
+from .recipes import Recipe
+
+
+def compute(recipe: Recipe) -> Part:
+    """Compute EKO component in isolation."""
+    return Part("ciao")
```

Comment:
Don't we need a better name here?!

Reply:
This part has to be implemented in #217 and further PRs. I actually started from there, and then I remembered I had to first fix the runcard.

Ignore every new file in `src/eko/runner/` for the time being. As soon as we close this PR, we will move the focus on them.

Reply:
okay could you plese then add some comments or TODOs otherwise it's difficult to understand. 
If I understand correctly #217 now contains a file changed which can be already included here, is this the case?

Reply:
You're right #217 is just an empty PR at the moment, so do not consider its content, I will reset the branch to this one as soon as we'll be ready to merge.

That is just the next one I'm going to work on, simply it happened to be historically the first one I opened (you can infer from the number) because of bad planning (i.e. I forgot I had to do this first).